styles/globals.css

```diff
@@ -27,6 +27,7 @@ body {
   max-width: 100vw;
   overflow-x: hidden;
   overflow-y: scroll;
+  font-size: 24px;
 }
 
 body {
```

Comment:
@DanielPopOut 
Normally your no scroll-bar would work, But those things conflict with the overflow attributes in body. 
Removed the overflow x and y will be enough to get rid of scroll-bar
![overflow attribute](https://github.com/DanielPopOut/dockScreen/assets/43502816/68d15f11-4185-4ac4-ae76-5f2c3fe94ddc)

Reply:
Good catch